README.md

```diff
@@ -190,6 +190,52 @@ Since the old audit log component does not suit our needs anymore, we introduce
 that you have to explicitly call with clojure maps, which get serialized to JSON and shipped somewhere.
 Currently there are S3 and HTTP shippers.
 
+#### HTTP
+
+The HTTP audit logger provides a log function that uploads events via HTTP(S),
+accessible through `(:log-fn your-audit-component)`. The function accepts a
+single argument, which should be a map representing the event you want to log,
+and returns nil.
+
+Set the `:api-url` key in the configuration to the endpoint of the Audit API
+that accepts events, and configure the name of the token to use for
+identification through `:token-name`, e.g.
+
+```clojure
+(audit/map->HTTP {:configuration {:api-url "https://audit.example.org/events"
+                                  :token-name :auditlog}})
+```
+
+Note that the HTTP audit logger needs access to the OAuth2 token refresher (see
+next section), configured with a token of the name specified above.
+
+### OAuth2 token refresher component
+
+The token refresher provides other componens with OAuth2 access tokens by means
```

Comment:
Too late ;) Is there a way I can quickly fix the typo? I guess I'll have to go through a new merge request...